doc/source/api/visualization/index.rst

```diff
@@ -55,55 +53,41 @@ environment. PyVista is then used to visualize the extracted data.
   contour2.field = "temperature"
   contour2.surfaces = ['symmetry', 'wall']
 
-  # copy
-  graphics_session1.Contours["contour-3"] = contour2()
-
-  # update
-  contour3 = graphics_session1.Contours["contour-3"]
-  contour3.update(contour1())
-
-  # delete
-  del graphics_session1.Contours["contour-3"] 
-
-  # loop
-  for name, _ in graphics_session1.Contours.items():
-      print(name)
-
   # iso surface
-  surface1.surface.iso_surface.field= "velocity-magnitude"
-  surface1.surface.iso_surface.rendering= "contour"
-
-  # display 
-  contour1.display()
-  mesh1.display()
-  surface1.display()
+  surface1.definition.type = "iso-surface"
+  surface1.definition.iso_surface.field= "velocity-magnitude"
+  surface1.definition.iso_surface.rendering= "contour"
+
+  # display
+  plotter = GraphicsWindow(grid=(2, 2))
+  plotter.add_graphics(mesh1, position=(0, 0))
+  plotter.add_graphics(contour1, position=(0, 1))
+  plotter.add_graphics(contour2, position=(1, 0))
+  plotter.add_graphics(surface1, position=(1, 1))
+  plotter.show()
   
-  # To display in specific window e.g. window-2
-  contour1.display("window-2")
-  
-Matplotlib example (XY plots)
------------------------------
+Visualization example (XY plots)
+--------------------------------
 
 Here plot data is extracted from the Fluent session into the Python
-environment. Matplotlib is then used to plot data.
+environment. Visualization is then used to plot data.
 
 .. code:: python
 
   # import module
-  from ansys.fluent.visualization import Plots
+  from ansys.fluent.visualization import XYPlot
 
   # get the plots object for the session
-  plots_session1 = Plots(solver_session)
-  
-  #get xyplot object
-  plot1=plots_session1.XYPlots["plot-1"]
+  plots_session1 = XYPlot(solver=solver_session)
```

Comment:
@prmukherj 
- Is this getting (as the comment states) or creating a plots object?
- Why is it a "plots" object? It seems to be singular (born out by the following code).
- The comment should be redundant if the code is doing as it appears
- THe name of the object seems unnecessarily complicated. Isn't it just a `plot`?